content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,180 @@ aliases:
+
+After running `influxd upgrade`, when first starting InfluxDB 2.0 the database generates time series index (TSI) files.
+(This may take some time depending upon the volume of data present.)
+
+## Before you begin: important considerations
+
+Before upgrading to InfluxDB 2.0, please consider the following guidelines;
+some or all might apply to your specific installation and use case.
+The sections below contain our recommendations for addressing possible gaps in the upgrade process.
+You may find while reading that you'd like to hold off on upgrading for now.
+
+### Continuous queries
+
+if you are currently using continuous queries, we recommend extracting them from your InfluxDB 1.x instance with `show continuous queries` prior to upgrade.
+Run the following:
+
+{{< keep-url >}}
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+1. Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+   Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+2. Kapacitor Stream-style TICKscripts will not work using the [subscription API]((/influxdb/v1.8/administration/subscription-management/)).
+   (There is no subscriptions API in InfluxDB 2.0.)
+   We recommend writing data directly to *both* Kapacitor and InfluxDB.
+   This allows stream tasks to continue to work.
+   To do this, configure two [InfluxDB output plugins](/telegraf/v1.16/plugins/#influxdb): one for Kapacitor and one for InfluxDB 2.0.
+   If you use other mechanisms to feed data into InfluxDB, Telegraf can be used as an intermediate layer to feed both Kapacitor and InfluxDB.
+
+   Other architectures and mechanisms possible, too.
+   Need more help? Reach out in the [community forums](https://community.influxdata.com).
+
+### User migrations
+
+`influxd upgrade` migrates existing 1.x users and their permissions.
+However, *administrative users are not migrated*.
+
+With the InfluxDB 1.x CLI, perform a `show users` query.
+Any users labeled "admin" will *not* be migrated.
+
+If using an admin user for visualization or Chronograf administrative functions, you may want to create a new read only user before upgrading.
```

Comment:
- If you're using...
- What do you mean by visualization here?
- read-only needs a hyphen
- might not may. And how do you know if you might want to?